app/models/space.rb

```diff
@@ -234,6 +254,14 @@ def to_text
     "
   end
 
+  def increment_exported_facts_count!
+    update_columns(exported_facts_count: exported_facts_count + 1)
+  end
+
+  def decrement_exported_facts_count!
+    update_columns(exported_facts_count: exported_facts_count - 1)
```

Comment:
let's not do this now, but we really shouldn't have to keep on using update_columns.